spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/DestinationNode.java

```diff
@@ -56,7 +56,7 @@ public String stringify(boolean includePositionalInfo) {
 
 	@Override
 	public String toString() {
-		return getDestinationName();
+		return ":" + getDestinationName();
```

Comment:
Do we really want to use `:` prefix for the destination name? Given, the `toString()` is expected to *only* return the actual destination name, we may not want to use the `:` prefix here as this could introduce some regression.